index.js

```diff
@@ -17,70 +19,111 @@ const productB = {
     commission: 75
 }
 
-// Light and dark mode switch
-const toggleBtn = document.querySelector('.toggle-mode')
-toggleBtn.addEventListener('click', function () {
-    document.body.classList.toggle('light')
-    toggleBtn.classList.toggle('fa-moon')
-    toggleBtn.classList.toggle('fa-sun')
-})
-
-// Setup
-// The sales buttons
-const paBtn = document.getElementById('btn-productA')
-const pbBtn = document.getElementById('btn-productB')
+// *** We have these emojis hardcoded in the html, so I don't believe these assignments are necessary - Matt
 // Show the correct emoji on those sales buttons
-paBtn.textContent = productA.emoji
-pbBtn.textContent = productB.emoji
-// Let them listen for clicks & when they're hit fire a sales function
-paBtn.addEventListener('click', function () { fixSale(productA) })
-pbBtn.addEventListener('click', function () { fixSale(productB) })
+// paBtn.textContent = productA.emoji
+// pbBtn.textContent = productB.emoji
 
-// Showing the sales results in the HTML
-const salesHeader = document.getElementById('sales-header')
-const salesBar = document.getElementById('sales-bar')
-const achievementsHeader = document.getElementById('achievements-header')
-const achievementsBar = document.getElementById('achievements-bar')
-// initally they're empty (need to refactor this when we're using local storage)
-salesBar.textContent = achievementsBar.textContent = ''
+// Fetch sales data from local storage (if available)
+function fetchSalesData() {
```

Comment:
Yes, essentially. Those variables you'd originally declared as lets were turned into the properties of my salesData object. I used an object to hold the data instead of an array because the key/value pair format offered better organization and more straightforward access of the individual data elements.